/**:
  ros__parameters:
    motor_ids: [1,2,11,3,4,13,5,6,15,7,8,17,9,10]
    motor_calib_offsets: [1472, 2064, 3071, 3058, 1009, 2040, 2000, 3073, 3080, 1047, 1027, 1018, 2037, 3020]
    encode_ticks_per_rad: 652
    max_encoder_value: 4095
    profile_velocity: 120
    profile_acceleration: 30